Makefile

```diff
@@ -14,6 +14,28 @@ PYPI_VERSION    := $(shell scripts/get-py-version)
 
 TESTPARALLELISM := 4
 
+OS := $(shell uname)
+EMPTY_TO_AVOID_SED := ""
```

Comment:
What? 😕

Reply:
The Makefile has `xyz` occurrences too, so adding just something in the middle of the `xyz` to break the string allows the task to not replace the search part of sed in lines 25 and 26.

it's a similar trick you can use with `ps` while "grepping" a process:

```bash
# This brings two process, the one you are looking for and the one that's searching for it.
$ ps aux | grep syslog
root                39   0.0  0.0  4395956   1196   ??  Ss   30Jul19   0:21.17 /usr/sbin/syslogd
fernandocarletti 15763   0.0  0.0  4277296    836 s003  S+    3:12PM   0:00.00 grep --color=auto syslog

# Just by adding an useless list operator, you can filter out the "search" process.
$ ps aux | grep 'syslo[g]' 
root                39   0.0  0.0  4395956   1196   ??  Ss   30Jul19   0:21.17 /usr/sbin/syslogd
```

So if we had `mv "cmd/pulumi-tfgen-xyz" cmd/pulumi-tfgen-${NAME}` in place of `mv "cmd/pulumi-tfgen-x${EMPTY_TO_AVOID_SED}yz" cmd/pulumi-tfgen-${NAME}`, the `pulumi-tfgen-xyz` would be replaced`.

Reply:
Sweet, thanks for the explanation!